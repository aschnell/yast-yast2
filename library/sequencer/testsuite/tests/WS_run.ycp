{

include "testsuite.ycp";
import "Sequencer";

define any f1() ``{ return "1"; }
define any f2() ``{ return 2; }
define any f3() ``{ return `id3; }

map aliases = $[
    "1" : ``(f1()),
    "2" : ``(f2()),
    "3" : ``(f3())
];

TEST(``(Sequencer::WS_run($[], "blah")), [], nil);
TEST(``(Sequencer::WS_run(aliases, "blah")), [], nil);
TEST(``(Sequencer::WS_run(aliases, "1")), [], nil);
TEST(``(Sequencer::WS_run(aliases, "2")), [], nil);
TEST(``(Sequencer::WS_run(aliases, "3")), [], nil);

}
