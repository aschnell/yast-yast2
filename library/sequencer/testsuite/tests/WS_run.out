Log	sequencer: Alias not found: blah
Log	sequencer: Bad id: blah
Return	nil
Log	sequencer: Alias not found: blah
Log	sequencer: Bad id: blah
Return	nil
Log	sequencer: Returned value not symbol: 1
Return	nil
Log	sequencer: Returned value not symbol: 2
Return	nil
Return	`id3
